modules/storage-persistent-storage-lifecycle.adoc

```diff
@@ -57,9 +57,18 @@ PVs by including `persistentVolumeClaim` in the Pod's volumes block.
 ifdef::openshift-origin,openshift-enterprise[]
 
 [id="pvcprotection_{context}"]
-== PVC protection
+== Storage Object in Use Protection
 
-PVC protection is enabled by default.
+The Storage Object in Use Protection feature ensures that PVCs in active use by a Pod and PVs that are bound to PVCs are not removed from the system, as this may result in data loss.
+
+Storage Object in Use Protection is enabled by default.
+
+[NOTE]
+====
+A PVC is in active use by a Pod when a Pod object exists that is using the PVC.
+====
+
+If a user deletes a PVC in active use by a Pod, the PVC is not removed immediately. PVC removal is postponed until the PVC is no longer actively used by any Pods. Also, if a cluster admin deletes a PV that is bound to a PVC, the PV is not removed immediately. PV removal is postponed until the PV is no longer bound to a PVC.
```

Comment:
s/in active/that is in active
s/immediately./immediately,

Reply:
Leaving `immediately.` to avoid run-on sentence.